backend/chainlit/db/__init__.py

```diff
@@ -1,9 +1,9 @@
 import os
 
-from chainlit.config import PACKAGE_ROOT, config
+from chainlit.config import BACKEND_ROOT, config
 from chainlit.logger import logger
 
-SCHEMA_PATH = os.path.join(PACKAGE_ROOT, "db/prisma/schema.prisma")
+SCHEMA_PATH = os.path.join(BACKEND_ROOT, "db/prisma/schema.prisma")
```

Comment:
Not sure if the slash could be a problem when running on Windows, here is a suggested change:
```suggestion
SCHEMA_PATH = os.path.join(BACKEND_ROOT, "db", "prisma", "schema.prisma")
```